### ISSUE TITLE
Android Microsoft Edge: Buttons, Form, Rescues

### ISSUE THREAD

**Issue by cieraclaire** [ixamxkamii/animal-rescue-website#10]
Buttons > Your buttons on my Android should have more padding or space between them on the sides and on the top and bottom for easier accessibility. On your adoption page, try and add space between your card and your Learn More buttons (maybe center them).

See if you can have one of your buttons link back to the home page. You Adoption page won't let me go back to the home page unless I select your logo which took me more than 5 seconds to figure out.

Form > Some of the boxes in your form section are too close together, some are also flushed to the left while other are off to the center more

Rescues > page missing
